cedar-testing/src/cedar_test_impl.rs

```diff
@@ -118,6 +177,22 @@ pub trait CedarTestImplementation {
         expected: Option<Value>,
     ) -> TestResult<bool>;
 
+    fn partial_is_authorized(
+        &self,
+        request: &Request,
+        entities: &Entities,
+        policies: &PolicySet,
+    ) -> TestResult<partial::FlatPartialResposne>;
+
+    fn partial_evaluate(
```

Comment:
Add a comment describing the bool return value, like the one for `interpret` above.